TESTING.md

```diff
@@ -214,6 +216,60 @@ automatically.
 * Chat responds with **potential issues** and **suggestions** like missing nullability checks or input sanitization (
   response may vary).
 
+### Edit
+
+#### Editing code
+<!-- unhide after full edit command is implemented -->
+<!-- - Verify that the option to run the `Edit Code` command is available from the sidebar, right-click menu, the command palette (option+c, /edit), or Option+K keyboard shortcut. -->
+1. Verify that the option to run the `Edit Code` command is available from the right-click menu.
+1. Highlight a section of code.
+1. Run the `Edit Code` command with instructions for how Cody should edit the selected code.
+1. Verify that Cody automatically applies edits to the selected code, per the user’s instructions.
+1. Verify that there is a notification that "Cody is working..." while edits are being applied.
+1. Verify that you can see a list of code lenses with a Cody icon above the generated code: `Show diff`, `Accept`, `Retry`, and `Undo`.
+<!-- 1. Verify that you can see a diff view of the edit in a new tab by clicking `Show diff`. -->
+1. Verify that you can prompt Cody to retry the command by clicking `Retry` and entering new instructions.
+1. Verify that you can undo the edit by clicking `Undo`.
+1. Verify that the ghost text disappears by clicking `Accept`.
+
+#### Expected behaviour
```

Comment:
This expected behavior is weak, it doesn't really describe what those buttons should *do*, which are peppered in the instructions. If you want to have "expected behavior" separate, let's break up the above test by each flow: Edit thru Accept; Edit &rarr; Undo; Edit &rarr; Show Diff; etc.